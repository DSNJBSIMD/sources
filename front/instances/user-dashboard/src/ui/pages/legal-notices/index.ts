export * from './legal-notices.page';
