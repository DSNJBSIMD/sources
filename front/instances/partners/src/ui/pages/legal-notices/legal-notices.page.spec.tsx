import { render } from '@testing-library/react';

import { LegalNoticesPage } from './legal-notices.page';

describe('LegalNoticesPage', () => {
  it('should match the snapshot', () => {
    // When
    const { container } = render(<LegalNoticesPage />);

    // Then
    expect(container).toMatchSnapshot();
  });
});
