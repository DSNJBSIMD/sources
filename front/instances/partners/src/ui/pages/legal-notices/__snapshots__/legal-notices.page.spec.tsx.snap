// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LegalNoticesPage should match the snapshot 1`] = `
<div>
  <div
    class="fr-col-12 fr-col-md-8"
  >
    <h1>
      Mentions légales
    </h1>
    <h2>
      Éditeur
    </h2>
    <p>
      Ce site est édité par le programme FranceConnect de la direction interministérielle du numérique (DINUM).
    </p>
    <h2>
      Directrice de publication
    </h2>
    <p>
      Linda Debernardi : Cheffe du pôle fédération d’identité des citoyens chez direction interministérielle du numérique (DINUM).
    </p>
    <h2>
      Conception et gestion du site
    </h2>
    <p>
      Ce site est développé en mode agile, selon un principe d’amélioration continue. Le suivi éditorial et graphique est assuré au quotidien par FranceConnect. Propriété intellectuelle et liens vers franceconnect.gouv.fr
    </p>
    <p>
      Tout site public ou privé est autorisé à établir, sans autorisation préalable, un lien (y compris profond) vers les informations diffusées sur franceconnect.gouv.fr.
    </p>
    <p>
      Sauf mention de propriété intellectuelle détenue par des tiers (notamment un crédit sur certaines images), les contenus de ce site sont proposés sous licence ouverte (licence MIT et licence AGPL-3.0).
    </p>
    <p>
      La mise en place de liens vers nos contenus et documents est cependant à privilégier par rapport à la reproduction sur un site tiers. La mention explicite du site source
       
      franceconnect.gouv.fr est recommandée.
    </p>
    <h2>
      Hébergement
    </h2>
    <p>
      Centre Informatique Douanier
      <br />
      Direction générale des douanes et droits indirects (DGDDI)
      <br />
      Siège : 11 rue des Deux-Communes 93558 Montreuil Cedex
    </p>
  </div>
</div>
`;
